samples/dashboards/demos/climate/demo.js

```diff
@@ -1,50 +1,159 @@
-let kpis = {};
-let darkMode = false;
-let temperatureScale = 'C';
-
-async function setupDashboard() {
-
-    citiesData = await buildCitiesData();
-    buildSymbols();
-
-    const cityConnector = await dataPool.getConnector(cityScope);
-    const map = await fetch(
-        'https://code.highcharts.com/mapdata/custom/world.topo.json'
-    ).then(response => response.json());
-    const mapPoints = await buildCitiesMap();
-
-    return new Promise(resolve => Dashboards.board('container', {
+// left arrow
+Highcharts.SVGRenderer.prototype.symbols.leftarrow = (x, y, w, h) => [
+    'M', x + w / 2 - 1, y,
+    'L', x + w / 2 - 1, y + h,
+    x - w / 2 - 1, y + h / 2,
+    'Z'
+];
+// right arrow
+Highcharts.SVGRenderer.prototype.symbols.rightarrow = (x, y, w, h) => [
+    'M', x + w / 2 + 1, y,
+    'L', x + w / 2 + 1, y + h,
+    x + w + w / 2 + 1, y + h / 2,
+    'Z'
+];
+
+const MathModifier = Highcharts.DataModifier.types.Math; // @todo switch module
+const RangeModifier = Dashboards.DataModifier.types.Range;
+
+const colorStopsDays = [
+    [0.0, '#C2CAEB'],
+    [1.0, '#162870']
+];
+const colorStopsTemperature = [
+    [0.0, '#4CAFFE'],
+    [0.3, '#53BB6C'],
+    [0.5, '#DDCE16'],
+    [0.6, '#DF7642'],
+    [0.7, '#DD2323']
+];
+
+setupBoard();
+
+async function setupBoard() {
+    let activeCity = 'New York',
+        activeColumn = 'TX',
+        activeSelection = [Date.UTC(2010, 0, 5), Date.UTC(2010, 11, 25)],
+        activeScale = 'C',
+        selectionTimeout = -1;
+
+    // Initialize board with most basic data
+    const board = await Dashboards.board('container', {
+        dataPool: {
+            connectors: [{
+                name: 'Range Selection',
+                type: 'CSV'
+            }, {
+                name: 'Cities',
+                type: 'CSV',
+                options: {
+                    csvURL: (
+                        'https://www.highcharts.com/samples/data/' +
+                        'climate-cities.csv'
+                    )
+                }
+            }]
+        },
+        editMode: {
+            enabled: true,
+            contextMenu: {
+                enabled: true,
+                icon: (
+                    'https://code.highcharts.com/gfx/dashboard-icons/menu.svg'
+                ),
+                items: [
+                    'editMode',
+                    {
+                        id: 'dark-mode',
+                        type: 'toggle',
+                        text: 'Dark mode',
+                        events: {
+                            click: function () {
+                                this.menu.editMode.board.container
+                                    .classList.toggle('highcharts-dark');
+                            }
+                        }
+                    }, {
+                        id: 'fahrenheit',
+                        type: 'toggle',
+                        text: 'Fahrenheit',
```

Comment:
This button doesn't work.

Reply:
For me it is working. Which of your expectation is not fulfilled?

Reply:
You are right, but it takes some time that is why I thought it doesn't work.